app/src/main/java/com/sunasterisk/a14day_challenge/ui/process/ProcessContract.kt

```diff
@@ -0,0 +1,15 @@
+package com.sunasterisk.a14day_challenge.ui.process
+
+import com.sunasterisk.a14day_challenge.data.model.User
+
+interface ProcessContract {
+
+    interface View {
+        fun updateRecyclerView(user: User, processList: ArrayList<String>)
+        fun moveToLoginScreen()
+    }
+
+    interface Presenter {
+        fun fetchDataToRecyclerView()
```

Comment:
Presenter không cần biết View có gì, đặt tên lại cho hợp lý 
```suggestion
        fun getProgresses()
```